components/text_normalization/requirements.txt

```diff
@@ -0,0 +1,3 @@
+git+https://github.com/ml6team/fondant.git@main
```

Comment:
```suggestion
fondant
```
As there was a new Fondant release, this should work

Reply:
We should keep the install form git, since our build and release pipelines expect it.

See https://github.com/ml6team/fondant/pull/238.

Reply:
Ok, will close this comment